mlflow/tensorflow/__init__.py

```diff
@@ -147,18 +143,18 @@ def log_model(
     rather than lists of length one. All other model output types are included as-is in the output
     DataFrame.
 
-    :param tf_saved_model_dir: Path to the directory containing serialized TensorFlow variables and
-                               graphs in ``SavedModel`` format.
-    :param tf_meta_graph_tags: A list of tags identifying the model's metagraph within the
-                               serialized ``SavedModel`` object. For more information, see the
-                               ``tags`` parameter of the
-                               ``tf.saved_model.builder.SavedModelBuilder`` method.
-    :param tf_signature_def_key: A string identifying the input/output signature associated with the
-                                 model. This is a key within the serialized ``SavedModel`` signature
-                                 definition mapping. For more information, see the
-                                 ``signature_def_map`` parameter of the
-                                 ``tf.saved_model.builder.SavedModelBuilder`` method.
+    When logging keras model, `keras_model` argument
+    must be set but `tf_saved_model_dir`, `tf_meta_graph_tags` and `tf_signature_def_key`
+    arguments must not be set.
```

Comment:
Can we update this block? `tf_saved_model_dir`, `tf_meta_graph_tags` and `tf_signature_def_key` don't exist anymore.